moveit_core/collision_detection_bullet/include/moveit/collision_detection_bullet/bullet_integration/basic_types.h

```diff
@@ -68,7 +68,7 @@ struct ContactTestData
   const std::vector<std::string>& active;
 
   /** \brief If after a positive broadphase check the distance is below this threshold, a contact is added. */
-  const double& contact_distance;
```

Comment:
This has the the potential to change semantics since a reference to this double is prone to be changed by the calling code.

Reply:
Reverted back.